### ISSUE TITLE
Pilotlog not tracking glider flights

### ISSUE THREAD

**Issue by thijsvdc** [sanhozay/PilotLog#20]
## Versions

PilotLog version : 2.0.0
Java version : openjdk 15.0.3 2021-04-20
Flightgear version : 2020.3.11
O/S : Ubuntu
O/S version : 21.10
Browser : FF
Browser version :

## Problem Description

My glider flight was not logged. After checking the logs, it seems that gliders are too ecological :)

```
Invalidated flight #613 because fuel has not decreased
Ended flight #613 in ask21-jsb from EBOS to EBOS, status INVALID
Deleted unresponsive flight #613 in ask21-jsb from EBOS to EBOS, status INVALID
```

**Comment by sanhozay** [comment]
Part of the reason I created this was to monitor fuel consumption for planning purposes and I didn't want that skewed by in-air refuelling -- hence the fuel check. I didn't consider gliders and I'm not sure how to consistently identify models that don't consume fuel. Perhaps changing the check to see if fuel hasn't increased would be adequate? After all, I don't really check if in-flight refuelling has happened: you could start with 10 gallons, consume five, add four and I would consider that valid.

**Comment by sanhozay** [comment]
Fixed in development, will be in version 2.1.0. The check for fuel decrease at the end of the flight is only made if the start fuel was greater than zero so this shouldn't prevent glider flights being logged.
